src/lib/feature-finder.js

```diff
@@ -19,15 +20,17 @@ export default function(cucumberOptions) {
   .then((files) => {
     files = _.flattenDeep(files);
     return files.map((file) => {
-      let featureData = parseFeature(file);
-      return featureData.children
+      let parsedData = parseFeature(file);
+      didDetectErrors = parsedData.didDetectErrors ? true : didDetectErrors;
```

Comment:
didDetectErrors is a boolean it looks like, you don't need to use bitwise here. You could simply `let {didDetectErrors} = parsedData`